packages/desktop-gui/cypress/fixtures/browsers.json

```diff
@@ -2,23 +2,23 @@
   {
     "name": "chrome",
     "displayName": "Chrome",
-    "family": "chrome",
+    "family": "chromium",
     "version": "50.0.2661.86",
     "path": "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
     "majorVersion": "50"
   },
   {
     "name": "chromium",
     "displayName": "Chromium",
-    "family": "chrome",
+    "family": "chromium",
     "version": "49.0.2609.0",
     "path": "/Users/bmann/Downloads/chrome-mac/Chromium.app/Contents/MacOS/Chromium",
     "majorVersion": "49"
   },
   {
-    "name": "canary",
-    "displayName": "Canary",
-    "family": "chrome",
+    "name": "chrome",
+    "displayName": "Chrome Canary",
```

Comment:
This will break our recent updates for the icons in both the Test Runner and the Dashboard as we are relying on this name being `Canary`